### PR TITLE
Opinionated suggestions

### PR DESCRIPTION
**Pull request by elliottinvent**
I submitted a [separate PR](https://github.com/bigskysoftware/hypermedia-systems/pull/9) for basic corrections, that I don't think are controversial. These suggestions are a bit more opinionated and includes restructuring a couple of sentences where I thought the meaning behind the sentences was difficult to grasp. I don't know if you guys want these kinds of suggestions (which is straying more into editing)? Feel free to ignore whatever you don't agree with, just my pov.

### CODE REVIEW

**Comment by elliottinvent** [comment]
I'm guessing this is failing because it's changes on top of the previous PR, and once they're merged it'll pass? Either that or I've screwed up the Asciidoc formatting somehow (I don't think so, was pretty careful with that). If you need me to fix something just shout.

**Review by dz4k — APPROVED** [review]

File: book/CH05_ExtendingHTMLAsHypermedia.adoc [hunk 1]
```diff
@@ -703,9 +703,9 @@ You might have noticed that the button was switched from a `GET` request to a `P
 htmx does _not_ include the closest enclosing form for `GET` requests, but it _does_ include the form for all other types
 of requests.
 
-This may seem a little strange, but it avoids junking up URLs that are used within form when dealing with history
-entries, which we will discuss in a bit.  And you can always include an enclosing form's values with an element that
-uses a `GET` by using the `hx-include` attribute, discussed next.
+This may seem a little strange, but this default behaviour avoids junking up URLs in history entries, which we will discuss
```
Comment (elliottinvent) [inline book/CH05_ExtendingHTMLAsHypermedia.adoc:706]: Apologies – I've used the British English spelling of behaviour / behavior

**Comment by elliottinvent** [comment]
Let me know if there's anything I need to do here.

Also, apologies but I've introduced some British spelling in `ln 706` of `book/CH05_ExtendingHTMLAsHypermedia.adoc`, I've added a comment to highlight it

**Comment by elliottinvent** [comment]
FYI, there's 300+ mentions of "behavior" in the book and two of "behaviour", both on `book/CH10_ScriptingInAHypermediaApplication.adoc` which also refers to a URL: 
https://htmx.org/essays/locality-of-behaviour/

I think if that's the spelling of the LoB principle then that's fine (Rule Britannia! 🇬🇧), but the page at that URL also has quite a bit of inconsistency too.

**Comment by adamstep** [comment]
> FYI, there's 300+ mentions of "behavior" in the book and two of "behaviour", both on `book/CH10_ScriptingInAHypermediaApplication.adoc` which also refers to a URL: https://htmx.org/essays/locality-of-behaviour/
> 
> I think if that's the spelling of the LoB principle then that's fine (Rule Britannia! 🇬🇧), but the page at that URL also has quite a bit of inconsistency too.

In the Hyperview chapters, `behavior` is a [keyword of the XML syntax](https://hyperview.org/docs/reference_behavior), so those should definitely not be changed! :)

**Comment by 1cg** [comment]
i have another big editing merge coming in.  Once that is in I'd like to merge this (or at least consider the language changes) can you ping me if this falls off my radar in a week?

**Comment by 1cg** [comment]
lol, but of course you know that, that's the big editing PR you submitted!

**Comment by elliottinvent** [comment]
@1cg since this isn't passing the checks (I submitted it on top of another commit, which was included in another PR by mistake), feel free to bin this PR and take whatever suggestions you like over into your next edit.

**Comment by elliottinvent** [comment]
@1cg a few of these issues have now been dealt with in @codetalcott edits. 

Also, since I submitted this on top of another PR by mistake, and introduced some British spelling, I'm going to simplify things by closing this PR and highlighting the remaining points that haven't been dealt with for @codetalcott consideration in his next edit. 

I'll raise the "locality of behavior" / "locality of behaviour" point as a separate issue.

**Pull request closed without merging**